lib/registration.pm

```diff
@@ -123,7 +123,7 @@ sub add_suseconnect_product {
     $version //= scc_version();
     $arch    //= get_required_var('ARCH');
     $params  //= '';
-    assert_script_run("SUSEConnect -p $name/$version/$arch $params");
+    assert_script_run("SUSEConnect -p $name/$version/$arch $params", 120);
```

Comment:
It's ok for us to increase timeout but you can also use SCALE_TIMEOUT just on your worker, also because the network contact can be slower